cc-backend/readme.md

```diff
@@ -21,3 +21,7 @@
 - nodeJS 16.14.0
 - typescript 4.8.3
 
+### Docker Guide 
```

Comment:
Maybe add note that you should add .env file before

Reply:
done